nexus/db-queries/tests/output/authz-roles.out

```diff
@@ -1,1577 +1,1838 @@
+  silo1-proj1-limited-collaborator  ✘  ✘  ✘  ✘  ✘  ✘  ✘  ✘
+  silo1-proj1-viewer                ✘  ✘  ✘  ✘  ✘  ✘  ✘  ✘
+  unauthenticated                   !  !  !  !  !  !  !  !
+  scim                              ✘  ✘  ✘  ✘  ✘  ✘  ✘  ✘
 
 resource: authz::AuditLog
 
-  USER                             Q  R LC RP  M MP CC  D
-  fleet-admin                      ✘  ✘  ✔  ✘  ✘  ✘  ✔  ✘
-  fleet-collaborator               ✘  ✘  ✔  ✘  ✘  ✘  ✔  ✘
-  fleet-viewer                     ✘  ✘  ✔  ✘  ✘  ✘  ✔  ✘
-  silo1-admin                      ✘  ✘  ✘  ✘  ✘  ✘  ✔  ✘
-  silo1-collaborator               ✘  ✘  ✘  ✘  ✘  ✘  ✔  ✘
-  silo1-viewer                     ✘  ✘  ✘  ✘  ✘  ✘  ✔  ✘
-  silo1-proj1-admin                ✘  ✘  ✘  ✘  ✘  ✘  ✔  ✘
-  silo1-proj1-collaborator         ✘  ✘  ✘  ✘  ✘  ✘  ✔  ✘
-  silo1-proj1-viewer               ✘  ✘  ✘  ✘  ✘  ✘  ✔  ✘
-  unauthenticated                  !  !  !  !  !  !  !  !
-  scim                             ✘  ✘  ✘  ✘  ✘  ✘  ✔  ✘
+  USER                              Q  R LC RP  M MP CC  D
+  fleet-admin                       ✘  ✘  ✔  ✘  ✘  ✘  ✔  ✘
+  fleet-collaborator                ✘  ✘  ✔  ✘  ✘  ✘  ✔  ✘
+  fleet-viewer                      ✘  ✘  ✔  ✘  ✘  ✘  ✔  ✘
+  silo1-admin                       ✘  ✘  ✘  ✘  ✘  ✘  ✔  ✘
+  silo1-collaborator                ✘  ✘  ✘  ✘  ✘  ✘  ✔  ✘
+  silo1-limited-collaborator        ✘  ✘  ✘  ✘  ✘  ✘  ✔  ✘
+  silo1-viewer                      ✘  ✘  ✘  ✘  ✘  ✘  ✔  ✘
+  silo1-proj1-admin                 ✘  ✘  ✘  ✘  ✘  ✘  ✔  ✘
+  silo1-proj1-collaborator          ✘  ✘  ✘  ✘  ✘  ✘  ✔  ✘
+  silo1-proj1-limited-collaborator  ✘  ✘  ✘  ✘  ✘  ✘  ✔  ✘
+  silo1-proj1-viewer                ✘  ✘  ✘  ✘  ✘  ✘  ✔  ✘
+  unauthenticated                   !  !  !  !  !  !  !  !
+  scim                              ✘  ✘  ✘  ✘  ✘  ✘  ✔  ✘
 
 resource: Silo "silo1"
 
-  USER                             Q  R LC RP  M MP CC  D
-  fleet-admin                      ✘  ✔  ✘  ✔  ✔  ✔  ✘  ✔
-  fleet-collaborator               ✘  ✔  ✘  ✔  ✔  ✔  ✘  ✔
-  fleet-viewer                     ✘  ✔  ✘  ✔  ✘  ✘  ✘  ✘
-  silo1-admin                      ✘  ✔  ✔  ✔  ✔  ✔  ✔  ✔
-  silo1-collaborator               ✘  ✔  ✔  ✔  ✘  ✘  ✔  ✘
-  silo1-viewer                     ✘  ✔  ✔  ✔  ✘  ✘  ✘  ✘
-  silo1-proj1-admin                ✘  ✔  ✘  ✔  ✘  ✘  ✘  ✘
-  silo1-proj1-collaborator         ✘  ✔  ✘  ✔  ✘  ✘  ✘  ✘
-  silo1-proj1-viewer               ✘  ✔  ✘  ✔  ✘  ✘  ✘  ✘
-  unauthenticated                  !  !  !  !  !  !  !  !
-  scim                             ✘  ✘  ✘  ✘  ✘  ✘  ✘  ✘
+  USER                              Q  R LC RP  M MP CC  D
+  fleet-admin                       ✘  ✔  ✘  ✔  ✔  ✔  ✘  ✔
+  fleet-collaborator                ✘  ✔  ✘  ✔  ✔  ✔  ✘  ✔
+  fleet-viewer                      ✘  ✔  ✘  ✔  ✘  ✘  ✘  ✘
+  silo1-admin                       ✘  ✔  ✔  ✔  ✔  ✔  ✔  ✔
+  silo1-collaborator                ✘  ✔  ✔  ✔  ✘  ✘  ✔  ✘
+  silo1-limited-collaborator        ✘  ✔  ✔  ✔  ✘  ✘  ✘  ✘
```

Comment:
Note: a `silo.limited_collaborator` can not create projects, as our project-create sagas add `default` VPCs and other networking resources during project creation.